src/include/duckdb/common/types/vector.hpp

```diff
@@ -410,6 +411,40 @@ struct StructVector {
 	DUCKDB_API static vector<unique_ptr<Vector>> &GetEntries(Vector &vector);
 };
 
+struct UnionVector {
+	static inline union_tag_t *GetTags(Vector &v) {
+		// the tag vector is always the first struct child.
```

Comment:
Can we move this to the source file similar to the others?

Reply:
Sure, they will not be inlined anymore but I think that's ok.